public/app/plugins/datasource/cloud-monitoring/components/LabelFilter.tsx

```diff
@@ -28,13 +28,20 @@ const filtersToStringArray = (filters: Filter[]) =>
 
 const operators = ['=', '!=', '=~', '!=~'].map(toOption);
 
+// These keys are not editable as labels but they have its own selector.
+// For example the 'metric.type' is set with the metric name selector.
+const protectedFilterKeys = ['metric.type'];
+
```

Comment:
this change is now needed since the metricType is now encoded as a filter not as a different field